### ISSUE TITLE
[bug] the scroll bar does not appear in box, when items are created dynamically with ngFor

### ISSUE THREAD

**Issue by 10229428** [zefoy/ngx-perfect-scrollbar#280]
In box 1, I click the 'update' button to generate some item by 'ngFor'. Then I mouseenter the box, but the scrollbar does not appear. I  have to operate the mouse wheel to show the scroll bar.

In box 2, some items are put in the box when the page is initialized. The scroll bar can display when I mouseenter the box.

I have a demo here:
https://stackblitz.com/edit/angular-ptzzn6

**Comment by 10229428** [comment]
Is anyone here?

**Comment by pedrojrivera** [comment]
I upgraded to 9.0.0 and experienced the very same issue.

scrollbar bar would work every where I had implemented it except for one spot where it was in an ngFor loop.

I downgraded to 8.0.0 and it works just fine.

9.0.0 is definitely broken.

**Comment by 10229428** [comment]
@pedrojrivera I got the issue in v8.0.0! You do not experience the bug in 8.0.0?

**Comment by pedrojrivera** [comment]
No...In 8.0.0 seems to work just fine. When I upgrade to 9.0.0 the very same code stops working.
